Getting-Started/Setup/Server-Setup/Load-Balancing/flexible.md

```diff
@@ -0,0 +1,102 @@
+#Flexible load balancing
+
+_Information on how to deploy Umbraco in a flexible Load Balanced scenario and other
+details to consider when setting up Umbraco for load balancing._
+
+**_Traditional load balancing must be used for Umbraco versions less than 7.3.0._**
+
+**Be sure you read the [Overview](index.md) before you begin!**
+
+##Design
+These instructions make the following assumptions:
+
+* All web servers can communicate with the database where Umbraco data is stored
+* You are running Umbraco 7.3.0 or above
+* _**You will designate a single server to be the back-office server for which your editors will log into for editing content.**_ Umbraco will not work if the back-office is behind the load balancer *(see DNS for more information below)*.
```

Comment:
No... it actually doesn't. It might seem to 'work fine' but there are all sort of things that can go wrong. In no way do we support load balancing the back office. If multiple admins are using the back office at the same time and things are being copied and moved and saved, things might end up with incorrect or non existent parents. We cannot guarantee the order that instructions are queued if you are load balancing the back office.